core/src/main/java/org/bitcoinj/core/Block.java

```diff
@@ -278,10 +278,10 @@ public int messageSize() {
         int size = HEADER_SIZE;
         List<Transaction> transactions = getTransactions();
         if (transactions != null) {
-            size += VarInt.sizeOf(transactions.size());
-            for (Transaction tx : transactions) {
-                size += tx.messageSize();
-            }
+            size += VarInt.sizeOf(transactions.size()) +
+                    transactions.stream()
+                            .mapToInt(Transaction::messageSize)
+                            .sum();
```

Comment:
I'll make a follow-on to this with the ternary (if you haven't already) for us to look at.

Reply:
https://github.com/bitcoinj/bitcoinj/pull/3666/commits/30101a7b0858e326263cf0e4f16758574cbb5720